lib/src/widgets/base_widgets.dart

```diff
@@ -161,7 +161,7 @@ class __SettingsTileState extends State<_SettingsTile> {
               widget.title,
               style: widget.titleTextStyle ?? headerTextStyle(context),
             ),
-            subtitle: widget.subtitle!.isEmpty
+            subtitle: widget.subtitle == null
```

Comment:
The String's `isEmpty` check is replaced with null check. Which is fine, but now if the String is actually an `empty` string then `Text` will be rendered. Correct?

Reply:
I added empty subtitle check.